app/uk/gov/hmrc/gform/sharedmodel/formtemplate/BooleanExpr.scala

```diff
@@ -75,6 +76,11 @@ object BooleanExpr {
         case _ => ("", List.empty)
       }
 
+    def multiValue(collectionFieldId: Expr): (List[String], List[FormComponentId]) = {
+      val (collection, fieldId) = stringValue(collectionFieldId)
```

Comment:
There is this `data: Map[FormComponentId, Seq[String]]` used in `stringValue`, and I think this ticket is great opportunity to amend previous errors of not embracing that type.

What I mean by this is that value of `Seq[String]` type should be holding all selections like `List("1", "2", "3")`, and not concatenated string of selections like List("1,2,3") as it currently does.